src/cartridges/app_adyen_SFRA/cartridge/client/default/js/adyen/checkout/index.js

```diff
@@ -247,14 +247,37 @@ function registerFirstPaymentMethod() {
 
 function init() {
   $(document).ready(async () => {
-    paymentMethodsResponse = await getPaymentMethods();
-    const storedCustomerEmail = sessionStorage.getItem('customerEmail');
-    $('body').trigger('checkout:renderPaymentMethod', {
-      email: storedCustomerEmail,
-    });
-    helpers.createShowConfirmationForm(
-      window.ShowConfirmationPaymentFromComponent,
-    );
+    let fastlane;
+    let watermarkContainer;
+    try {
+      paymentMethodsResponse = await getPaymentMethods();
+      const paymentMethods =
+        paymentMethodsResponse?.AdyenPaymentMethods?.paymentMethods;
+      const isFastlaneEnabled = paymentMethods?.some(
+        (pm) => pm.type === 'fastlane',
+      );
+      if (isFastlaneEnabled) {
+        const guestEmail = document.querySelector('#email-guest');
+        if (guestEmail) {
```

Comment:
Here the only way we are identifying the guest shopper flow is using the "guestEmail". Should we also check if the shopper is logged in or not to determine the guest shopper flow?